cmd/snap-update-ns/change.go

```diff
@@ -87,7 +88,23 @@ func mimicRequired(err error) (needsMimic bool, path string) {
 	return false, ""
 }
 
-func (c *Change) createPath(path string, pokeHoles bool, as *Assumptions) ([]*Change, error) {
+// TODO: Consider if this can be replaced or become osutil function
+// Created this to ensure links are not followed and error is reported.
+func dirExists(path string) (bool, error) {
```

Comment:
We have `osutil.DirExists()` - would that work as well instead of needing another helper?

Reply:
Unlike the current osutil version, this version of dirExists() does not follow symlinks, which aligned with the existing snap-update-ns "ensure target" mechanism.